.github/workflows/test_github_action.yml

```diff
@@ -26,6 +25,18 @@ jobs:
       with:
         v-version: ${{ matrix.v-version }}
       id: v
-
+      
     - name: Test V is present
       run: v version
+      
+    - name: Test V fmt
```

Comment:
Why do we need to test V commands here? It's enough to run `v version` and `${{ steps.v.outputs.v_home }}/v version` to make sure V executable is visible.

Reply:
Because I've got one people where the command `v fmt -verify` does not work when using the action.